mypy/main.py

```diff
@@ -203,6 +204,16 @@ def add_invertible_flag(flag: str,
             strict_flag_names.append(flag)
             strict_flag_assignments.append((dest, not default))
 
+    def disallow_any_argument_type(raw_options: str) -> List[str]:
+        current_options = raw_options.split(',')
```

Comment:
Good point. I was trying to make it clear that these options are the ones provided by user. I think just `options` works too!